modules/core/kivent_core/managers/animation_manager.pyx

```diff
@@ -0,0 +1,33 @@
+from kivent_core.managers.resource_managers cimport ModelManager
+from kivent_core.managers.resource_managers import texture_manager
+from kivent_core.memory_handlers.block cimport MemoryBlock
+from kivent_core.rendering.animation cimport FrameList
+
+
+cdef class AnimationManager:
+
+    def __init__(self, ModelManager model_manager, allocation_size=1024*10):
+        self.model_manager = model_manager
+        self.allocation_size = allocation_size
+        self.memory_block = None
+        self._animations = {}
+
+    def allocate(self, master_buffer, config_dict):
+        cdef MemoryBlock memory_block = MemoryBlock(self.allocation_size, 1, 1)
```

Comment:
Yep!

```#1  0x00007fffe7ea8630 in __pyx_f_11kivent_core_15memory_handlers_5block_11MemoryBlock_allocate_memory_with_buffer (__pyx_v_self=0x7fffd9fdccd0, __pyx_v_master_buffer=0x8fb4d0 <_Py_NoneStruct>)
    at kivent_core/memory_handlers/block.c:1162```

Notice how the master buffer is actually a _Py_NoneStruct, at least if I'm reading this right that was the tip off that made me go look for the None